CGPs/template.md

```diff
@@ -0,0 +1,41 @@
+# CIP [0000]: Title for Governance Proposal
+
+- Date: 2020-05-11
+- Author: @githubhandle
+- Status: DRAFT|PROPOSED|EXECUTED|EXPIRED
+	- DRAFT = Feedback collection
+	- PROPOSED = The governance proposal has been submitted to the network
+	- EXECUTED = The governance proposal was executed on the network
+	- EXPIRED = The governance proposal expired
+- Governance Propsal ID #: [if submitted]
+- Date Executed: [if executed]
+
+## Overview
+
+Pick a four digit number subsequent to the latest PR raised and name your file that.
+
+Describe the issue that motivates this CGP. It should indicate all parameters that are being changed and why doing so is important.
+
+Explain what benefits the enhancement this change will bring. To the extent possible, enumerate use cases affected by this CGP.
+
+## Proposed Changes
+
+Fill out the following template for each transaction in the proposal
+
+1. Description of transaction 1 
```

Comment:
Is the "1" after transaction supposed to be there?

Reply:
Yup